docs/speech-ui-api-docs.md

```diff
@@ -0,0 +1,83 @@
+# Commands to get the UI and server running once configured
+```
+sh target/appassembler/bin/PyseriniEntryPoint
+python3 src/main/python/api.py
```

Comment:
Starting it with the root directory as the working directory doesn't actually work with the config - I'll fix it as a part of #158.